server/events/runtime/run_step_runner.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // RunStepRunner runs custom commands.
 type RunStepRunner struct {
-	TerraformExecutor TerraformExec
+	TerraformExecutor StepCmdExec
```

Comment:
Why did you change this? We should minimize changes and this doesn't seem super essential to do now imo.